assets/templates/partials/json-ld/dataset/legacy.tmpl

```diff
@@ -1,16 +1,17 @@
 {{ $data := .DatasetLandingPage }}
+{{ $URLPath := .URI }}
     {{ $length := len $data.Datasets }}{{ if gt $length 0 }}
         {{ $latestDataset := index $data.Datasets 0}}
         {{ $numberOfDownloads := len $latestDataset.Downloads }}
         "distribution": [
         {{ range $i, $el := $latestDataset.Downloads }} {
             "@type": "DataDownload",
             "encodingFormat": {{ .Extension }},
-            "contentUrl": {{ .URI }}
+            "contentUrl": {{concatenateStrings "www.ons.gov.uk/file?uri=" $URLPath "/current/" .URI }}
             }{{ if notLastItem $numberOfDownloads $i }},{{end}}
         {{ end }}
     ],
     {{ end }}
-    "url": {{ buildURL .URI .SiteDomain }},
+    "url": {{ concatenateStrings "www." .SiteDomain $URLPath }},
```

Comment:
When on welsh pages this should probably be using the `cy.` subdomain rather than the `www.` subdomain.

Reply:
Updated